notification-portlet-webcomponents/notification-icon/README.md

```diff
@@ -0,0 +1,29 @@
+# notification-icon
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
```

Comment:
You don't use a `npm run start` like in uPortal-webcomponents ?

Reply:
Vue CLI, uses `serve` by default, I'll update it to bring it inline with uPortal web components.

Reply:
Updated